components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/AccessTokenIssuerTest.java

```diff
@@ -1,770 +1,300 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

Comment:
2017-2025

Reply:
This file is newly created. The existing one was renamed as DeprecatedAccessTokenISsuerTest